src/Template/Element/FilterBox/filter_box_common.twig

```diff
@@ -66,12 +66,21 @@
             </label>
         </span>
 
-        <span v-if="filter.date">
+        <span v-if="filter.name === 'date_ranges'">
+            <label>{{ __('From date') }}
+            <input-dynamic-attributes :value.sync="queryFilter.filter['date_ranges']['from_date']" :attrs="filter" />
+            </label>
+            <label>{{ __('To date') }}
+            <input-dynamic-attributes :value.sync="queryFilter.filter['date_ranges']['to_date']" :attrs="filter" />
+            </label>
+        </span>
+
+        <span v-if="filter.date && (filter.name !== 'date_ranges')">
```

Comment:
maybe this can be shortened with `v-else-if="filter.date"`

Reply:
you're right, but we should probably apply `v-else-if` to previous  `<span>` as well and add a final `v-else` ?

Reply:
yes `v-else-if` is ok on the second span checking `filter.type`'s value. final `v-else` is ok if the value of `filter.date` isn't relevant to show its content

Reply:
ok, have look now